test/admin/draft.test.js

```diff
@@ -0,0 +1,41 @@
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+
+import { buildAdminTestHarness, extractTestFixtureDataPoints } from '../testutil';
+
+describe('Draft', () => {
+  let test = new StitchMongoFixture();
+  let th;
+  let appDraft;
+
+  beforeAll(() => test.setup());
+  afterAll(() => test.teardown());
+
+  beforeEach(async() => {
+    const { apiKey, groupId, serverUrl } = extractTestFixtureDataPoints(test);
+    th = await buildAdminTestHarness(true, apiKey, groupId, serverUrl);
+    appDraft = th.app().draft();
+  });
+
+  afterEach(async() => th.cleanup());
+
+  it('responds with a 404 when attempting to get a draft that does not exist', async() => {
+    try {
```

Comment:
should use `expect.assertions` in code blocks like this one. otherwise if it *doesn't* 404, then the `catch` block never executes and the assertions never get hit, so the test will silently pass even though the code didn't do what it's supposed to do.